scipy `DeprecationWarning`  in`scipy.signal.remez`: use keyword arguments instead of positional

from https://github.com/isce-framework/dolphin/actions/runs/7615612433/job/20740529928?pr=197

```
tests/test_unwrap.py:161: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/dolphin/unwrap/_unwrap.py:228: in unwrap
    return multiscale_unwrap(
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/dolphin/unwrap/_tophu.py:165: in multiscale_unwrap
    tophu.multiscale_unwrap(
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/tophu/_multiscale.py:647: in multiscale_unwrap
    coarse_unwrapped, coarse_conncomp = coarse_unwrap(
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/tophu/_multiscale.py:339: in coarse_unwrap
    igram_lores = lowpass_filter_and_multilook(
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/tophu/_multiscale.py:108: in lowpass_filter_and_multilook
    coeffs2d = np.outer(*coeffs1d)
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/tophu/_multiscale.py:97: in get_filter_coeffs
    return bandpass_equiripple_filter(
../../../micromamba/envs/dolphin-env/lib/python3.11/site-packages/tophu/_filter.py:173: in bandpass_equiripple_filter
    coeffs = scipy.signal.remez(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (39, [0.0, 0.13333333333333333, 0.2, 0.5], [1.0, 0.0], [0.01, 0.0011506300634948063])
kwargs = {'fs': 1.0, 'grid_density': 16, 'maxiter': 25}, extra_args = 1
args_msg = 'weight=[0.01, 0.0011506300634948063]'

    @functools.wraps(f)
    def inner_f(*args, **kwargs):
        extra_args = len(args) - len(all_args)
        if extra_args <= 0:
            return f(*args, **kwargs)
    
        # extra_args > 0
        args_msg = [
            f"{name}={arg}"
            for name, arg in zip(kwonly_args[:extra_args], args[-extra_args:])
        ]
        args_msg = ", ".join(args_msg)
>       warnings.warn(
            (
                f"You are passing {args_msg} as a positional argument. "
                "Please change your invocation to use keyword arguments. "
                f"From SciPy {version}, passing these as positional "
                "arguments will result in an error."
            ),
            DeprecationWarning,
            stacklevel=2,
        )
E       DeprecationWarning: You are passing weight=[0.01, 0.0011506300634948063] as a positional argument. Please change your invocation to use keyword arguments. From SciPy 1.14, passing these as positional arguments will result in an error.

```

--- a/src/tophu/_filter.py
+++ b/src/tophu/_filter.py
@@ -171,10 +171,10 @@ def bandpass_equiripple_filter(
 
     # Get filter coefficients.
     coeffs = scipy.signal.remez(
-        numtaps,
-        edgefreqs,
-        gains,
-        weights,
+        numtaps=numtaps,
+        bands=edgefreqs,
+        desired=gains,
+        weight=weights,
         maxiter=maxiter,
         grid_density=grid_density,
         fs=samplerate,

